aries-site/src/examples/components/search/SearchExample.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import styled from 'styled-components';
+import { TextInput, Box } from 'grommet';
+import { Search as SearchIcon } from 'grommet-icons';
+
+// Inputs should always be accompanied by labels for accessibility. An icon
```

Comment:
Im glad this is within each example so they can see that this is needed each time!